WeakAuras/Prototypes.lua

```diff
@@ -3739,10 +3743,22 @@ Private.event_prototypes = {
         conditionType = "string"
       },
       {
+        name = "spellSchool",
+        display = L["Spell School"],
+        type = "select",
+        values = "combatlog_spell_school_types_for_ui",
+        test = "spellSchool and bit.band(spellSchool, %q) ~= 0",
```

Comment:
That doesn't work, in two ways:

a) E.g. a check for holy, also will match a "holyfire" event. (Or is that what the user expects?). That would make e.g. Cosmic events match "every" filter.
b) A check for "holyfire", also will match a "holy" event. That's clearly not right. 

So I think we actually might want to have state.spellSchool == %q ?

Reply:
let's go for spellSchool == %d